base.sh

```diff
@@ -38,12 +45,10 @@ job_pids_cache=""
 # which may cause next job to be submitted to wrong cluster
 unset %(CSUB_SERVER)s
 
-# make sure qsub is available
-module load scripts
-
 jobname=${%(CSUB_JOBNAME)s}
 ## jobname_stripped is set in BASEHEADER
 scriptname="${jobname_stripped}.sh"
+script_pid=0
```

Comment:
why default a pid that always exists?

Reply:
hmm, ok, I'll just drop this (`kill -0` without an argument exits with `1`)